documentation/modules/ROOT/pages/03-demo.adoc

```diff
@@ -7,28 +7,879 @@
+
+////
+Online editor:
+https://www.plantuml.com/plantuml/uml/VP1DJiCm58JtFiMZ-rmWYwgqeHkeX2WNUBK7Ok4ubdyYzVQuZKbe5TZ5olTcFiqcHFOnTKOyn1OTIC8d0xPLdwBH5iBb_rfgnpRIwWMVBC_qwDoAED3ul4MUBKSzW9u6vES1eRsYMzz_mT-YZS-W3tJeLUwyOdlW23zeYJkK8vyuZ52p5O9bRk687uTYLgrB4zNqcav6XvPsR6GocTsZQ8d2L1aV3slQzVP3-uuKpCNgB1JkEwQpzI_FcjxoL5XgcUvdMioVL4soi-iuIOQcE5N259RYPgKYMNJ-3lfdkMPRqp7s7lJkjQFBvWihR61Lwimt
+
+@startuml
+!theme materia-outline
+participant "Legacy App" as Legacy
+participant "Knative _Event Mesh_" as Broker
+participant "Drivers Module" as FeeService
+participant "Database" as DB
+
+activate Legacy
+Legacy -> Broker : Publish CalculateFee Event
+Broker --> Legacy: Confirm delivery
+deactivate Legacy
+
+Broker -> FeeService: Route CalculateFee Event
+activate FeeService
+FeeService --> Broker: Publish DriverFeeCalculated Event
+deactivate FeeService
+
+Broker -> Legacy: Route DriverFeeCalculated Event
+activate Legacy
+Legacy -> DB: Store Trip Data
+deactivate Legacy
+@enduml
+////
+
+The diagram illustrates the flow of events between the legacy application, the Knative _Event Mesh_, the fee calculator service, and the datastore.
+
+Next, you can learn how to walk through this demo.
 
 == Run the demonstration
 
 === Before getting started
-To run this demo, you will need xpto. Adding to that, make sure to have:
 
-* ABC
-* XYZ
-* XPTO
+We'll be using the Red Hat OpenShift Container Platform (OCP) 4.x cluster, so make sure you have it available in your environment.
+
+You could use the https://developers.redhat.com/products/openshift/overview[Red Hat's Developer Sandbox] to spin up an instance for you.
+
+Alternatively, you can use the https://developers.redhat.com/products/openshift-local/overview[OpenShift Local] installation.
+Make sure to give it enough resources to fit the Serverless Operator and our demo application.
 
 === Installing the demo
-Installation guide and basic test of the demo installation if needed
+
+==== Installing the Serverless Operator
+
+To install the Serverless Operator, follow https://docs.openshift.com/serverless/1.35/install/preparing-serverless-install.html[the documentation steps].
+
+The *TL;DR* version would be to apply the following manifest, and wait until the operator is ready:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-serverless
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-serverless
+  namespace: openshift-serverless
+spec: {}
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: serverless-operator
+  namespace: openshift-serverless
+spec:
+  channel: stable
+  name: serverless-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+
+Here are commands to apply the above manifests.
+
+[.console-input]
+[source,shell]
+----
+git clone https://github.com/cardil/cabs-usvc
```

Comment:
we have a corp repo, instead of personal ones?

Reply:
Sure, I can push somewhere into a shared space. The repo is earlier than this solution, and it was the demo part of [my talk](https://youtu.be/Rc5IO6S6ZOk), so that's why it's there.

I think, I saw other solutions are okay with using personal repos for examples, but I'll check that out with @jayachristina.

Reply:
I got a response. It would be better, if we push the examples to some shared space. I proposed https://github.com/openshift-knative

Reply:
Ok, I just noticed a problem with this :cold_sweat:. See, here: https://github.com/legacyfighter/cabs-java/issues/2

([my example repo](https://github.com/cardil/cabs-usvc) is properly licensed, but it uses [this fork](https://github.com/cardil/legacyfighter-cabs-java/tree/bf576d61b679fd7e60b8668cb3bff7333e54fb7f), which doesn't have the proper license)

Reply:
ok. interesting